tests/test_dolt.py

```diff
@@ -62,6 +62,49 @@ def create_test_table(init_empty_test_repo, create_test_data) -> Tuple[Dolt, str
         _execute(["table", "rm", "test_players"], repo.repo_dir)
 
 
+@pytest.fixture
+def test_repo_with_two_remote_branches(
+    empty_test_repo_with_remote,
+) -> tuple[Dolt, str, str, str]:
```

Comment:
tuple -> Tuple is the only blocker for mer running this locally

Reply:
oh yes, I am already on 3.10, but I see you support from 3.6. Changed.